engine/src/main/java/org/hibernate/search/engine/metadata/impl/TypeMetadata.java

```diff
@@ -234,6 +240,20 @@ public BridgeDefinedField getBridgeDefinedFieldMetadataFor(String fieldName) {
 		return null;
 	}
 
+	public FacetMetadata getFacetMetadataFor(String facetFieldName) {
+		FacetMetadata result = facetFieldNameToFacetMetadata.get( facetFieldName );
+		if ( result != null ) {
+			return result;
+		}
+		for ( EmbeddedTypeMetadata element : embeddedTypeMetadata ) {
```

Comment:
Is there a reason why we do not include the embedded metadata in the various maps?